### ISSUE TITLE
Remove or repair bogus style tests from tests/wpt/mozilla

### ISSUE THREAD

**Issue by emilio** [servo/servo#12886]
#12572 uncovered a few tests that weren't up to date with the specs. We should remove them or disable them once it lands.

cc #12758 

This means we should either remove them, or make them test the actual spec, and update the expectations.

Will update back with the actual filenames  once it lands.

**Comment by Coder206** [comment]
@emilio I would like to work on this.

**Comment by emilio** [comment]
It's yours then! :)

**Comment by Coder206** [comment]
@emilio Thanks :)

**Comment by Coder206** [comment]
@emilio Are these the bogus tests you are referencing from above?

`▶ FAIL [expected PASS] /css-flexbox-1_dev/html/flex-flexitem-percentage-prescation.htm
  └   → /css-flexbox-1_dev/html/flex-flexitem-percentage-prescation.htm 549dc2995e7de786feb6065fc8fc76b53746d920
/css-flexbox-1_dev/html/reference/flex-flexitem-percentage-prescation-ref.htm 516cbcf7b447cf7d5a568a4f8b57f030555e6525
Testing 549dc2995e7de786feb6065fc8fc76b53746d920 == 516cbcf7b447cf7d5a568a4f8b57f030555e6525

  ▶ FAIL [expected PASS] /css21_dev/html4/run-in-listitem-between-001.htm
  └   → /css21_dev/html4/run-in-listitem-between-001.htm ef608dda75ea29fbec3a0a32f64cc80292e62d82
/css21_dev/html4/reference/run-in-text-ref.htm d068883ccfce1b3fdf24f9ea6b401c9a047b593c
Testing ef608dda75ea29fbec3a0a32f64cc80292e62d82 == d068883ccfce1b3fdf24f9ea6b401c9a047b593c

  ▶ FAIL [expected PASS] /css21_dev/html4/run-in-listitem-between-002.htm
  └   → /css21_dev/html4/run-in-listitem-between-002.htm ef608dda75ea29fbec3a0a32f64cc80292e62d82
/css21_dev/html4/reference/run-in-text-ref.htm d068883ccfce1b3fdf24f9ea6b401c9a047b593c
Testing ef608dda75ea29fbec3a0a32f64cc80292e62d82 == d068883ccfce1b3fdf24f9ea6b401c9a047b593c`

**Comment by Coder206** [comment]
@emilio So just fixing #12758?

**Comment by Coder206** [comment]
@emilio Could you please remove the C-Assigned tag to let someone else work on this. I am getting caught up in issues.

**Comment by jdm** [comment]
Removing the `E-easy` tag until there's more description of what's required here.

**Comment by emilio** [comment]
Fair enough, sorry for not being able to provide more details in this bug. I'll try to take the time today while at the airport to do it.

Also, there's a WPT test that needs an update, I'll split that into a separate issue.
